apps/server/src/infra/runtime-config/repo/runtime-config.repo.ts

```diff
@@ -0,0 +1,98 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { BaseDomainObjectRepo } from '@shared/repo/base-domain-object.repo';
+import { RuntimeConfigDefault, RuntimeConfigValue, RuntimeConfigValueAndType } from '../domain/runtime-config-value.do';
+import { RuntimeConfigEntity } from './entity/runtime-config.entity';
+import { RuntimeConfigRepo } from '../domain/runtime-config.repo.interface';
+import { EntityData, EntityName } from '@mikro-orm/core/typings';
+import { EntityManager, ObjectId } from '@mikro-orm/mongodb';
+import { RuntimeConfigValueFactory } from './runtime-config-value.factory';
+import { RuntimeConfigValueInvalidDataLoggable } from '../domain/loggable/runtime-config-invalid-data.loggable';
+import { RUNTIME_CONFIG_DEFAULTS } from '../injection-keys';
+
+@Injectable()
+export class RuntimeConfigMikroOrmRepo
+	extends BaseDomainObjectRepo<RuntimeConfigValue, RuntimeConfigEntity>
+	implements RuntimeConfigRepo
+{
+	constructor(
+		protected readonly _em: EntityManager,
+		@Inject(RUNTIME_CONFIG_DEFAULTS) private defaults: RuntimeConfigDefault[]
+	) {
+		super(_em);
+	}
+
+	get entityName(): EntityName<RuntimeConfigEntity> {
+		this._em.getRepository(RuntimeConfigEntity);
+		return RuntimeConfigEntity;
+	}
+
+	public async getAll(): Promise<RuntimeConfigValue[]> {
+		const entities = await this._em.find(RuntimeConfigEntity, {});
+		const entityMap = new Map(entities.map((e) => [e.key, e]));
+		const values = this.getAllConfigDOsDefinedInDefaults(entityMap);
+
+		return values;
+	}
+
+	private getAllConfigDOsDefinedInDefaults(entityMap: Map<string, RuntimeConfigEntity>): RuntimeConfigValue[] {
```

Comment:
The name is too long and I don't like DO in it. Furthermore it tries to explain something that is to complicated. I suggest:

```suggestion
	private getAllWithValidKey(entityMap: Map<string, RuntimeConfigEntity>): RuntimeConfigValue[] {
```